tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -4043,6 +4043,43 @@ public function test_add_converted_currency_information_appends_converted_curren
+		);
+		WC_Stripe_Database_Cache::set( 'checkout_session_' . $checkout_session_id, $checkout_session );
+
+		$formatted_total = '$10.00';
+		$result          = $this->mock_gateway->add_converted_currency_information( $formatted_total, $order );
+		$expected_amount = WC_Stripe_Helper::get_woocommerce_amount_from_stripe_amount( 1500, 'eur' );
+
+		WC_Stripe_Database_Cache::delete( 'checkout_session_' . $checkout_session_id );
+		remove_filter( 'woocommerce_is_account_page', '__return_true' );
+
+		$this->assertEquals( '$10.00 (&euro; ' . $expected_amount . ' EUR)', $result );
+	}
```

PHPMD (2.15.0)

[error] The method test_add_converted_currency_information_appends_converted_currency_info_on_my_orders_page is not named in camelCase. (undefined)

(CamelCaseMethodName)